doc-content/jbpm-docs/src/main/asciidoc/BPMN2/appe-process-elements.adoc

```diff
@@ -1078,7 +1078,7 @@ The Escalation End Event finishes the incoming workflow, that means consumes the
 [[_terminate_end_event]]
 ===== Terminate End Event
 
-The Terminate End Event finishes all execution flows in the specified process instance. Activities being executed are canceled. If a Terminate End Event is reached in a sub-process, the entire process instance is terminated.
+The Terminate End Event finishes all execution flows in the specified process instance. Activities being executed are canceled. The subprocess instance is terminated if a terminate end event is reached in a subprocess.
```

Comment:
Minor suggestion - Can it be simplified further?

s/The subprocess instance is terminated if a terminate end event is reached in a subprocess./The process instance stops immediately if a terminate end event is reached.

(or subprocess)